QPolygonTransform_lib/src/Transformation.cpp

```diff
@@ -11,14 +12,19 @@ Transform::Transform(const QPointF& rotCenter, float deg_angle)
 
 QPointF Transform::operator()(const QPointF& point)
 {
-    float a_angle = atan2(point.y(), point.x()); // calculation angle
-    float a_module =
-        (point.x() - m_rotCenter.x()) / cosf(a_angle); // calculation module
-    qDebug() << a_module;
 
-    m_angle += a_angle;
+    QPointF new_point = point - m_rotCenter;
+    std::complex<float> new_point_cpl(new_point.x(), new_point.y());
 
-    QPointF a_point{a_module * cos(m_angle) + m_rotCenter.x(), // calc new point
-                    a_module * sin(m_angle) + m_rotCenter.y()};
-    return a_point;
+    float angle = std::arg(new_point_cpl);
+    if (new_point.y() < 0)
+    {
+        angle += 6.283;
```

Comment:
Please consider extracting `pi` constant to separate file as `constexpr double`

E.g.
```cpp
constexpr double pi =  3.141592653589793238462643383279502884197169399375L;

constexpr double two_pi = 2*pi;
constexpr double inv_pi = 1 / pi;
```